server.py

```diff
@@ -268,14 +268,15 @@ def initialize_mysql(test=False):
             conn = engine.connect()
         else:
             conn = connect_mysql('root', '', '127.0.0.1', 3306, 'nyudevops')
-
+            
+debug = (os.getenv('DEBUG', 'False') == 'True')
+port = os.getenv('PORT', '5000')
```

Comment:
@rohithreddy1, are these supposed to be class variables? If so, can you move them to the top. It looks like they are part of the initialize_mysql method here.